index.html

```diff
@@ -200,6 +211,16 @@ <h1 class="title is-1"><a name="showcase">#</a> Showcase</h1>
     </script>
 
     <script>
+        // List of bindings; Edit this list to add or remove projects
+        const bindings_list = [
+            {
+                "name": "wgpu4k",
+                "description": "Work-in-progress binding for Kotlin/Multiplatform",
+                "website": "https://GitHub.com/wgpu4k/wgpu4k",
+                "thumbnail": "https://avatars.githubusercontent.com/u/163670885"
+            }
+        ]
```

Comment:
I don't think we need a thumbnail, just a boring list would be okay

Reply:
With the fancy link box or just a bullet list?

Reply:
Just bullet list is fine